common/src/main/java/com/wynntils/commands/MenuCommand.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © Wynntils 2022-2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.commands;
+
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.context.CommandContext;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.consumers.commands.Command;
+import com.wynntils.screens.base.WynntilsMenuScreenBase;
+import com.wynntils.screens.wynntilsmenu.WynntilsMenuScreen;
+import net.minecraft.commands.CommandSourceStack;
+
+public class MenuCommand extends Command {
+    @Override
+    public String getCommandName() {
+        return "menu";
+    }
+
+    @Override
+    protected LiteralArgumentBuilder<CommandSourceStack> getCommandBuilder(
+            LiteralArgumentBuilder<CommandSourceStack> base) {
+        return base.executes(this::openMenu);
+    }
+
+    private int openMenu(CommandContext<CommandSourceStack> commandSourceStackCommandContext) {
+        Managers.TickScheduler.scheduleLater(() -> WynntilsMenuScreenBase.openBook(WynntilsMenuScreen.create()), 2);
```

Comment:
Why the delay?

Reply:
Okay, I see. Perhaps copy the comment too..

Reply:
> overwriting the lootrun screen

Maybe change that to be more accurate for the screen it will open :)